source/_components/solaredge.markdown

```diff
@@ -46,75 +48,19 @@ name:
-  description: SolarEdge Site information to be monitored. The following conditions can be monitored.
-  required: false
-  type: list
-  default: current_power
-  keys:
-    lifetime_energy:
-      description: Lifetime energy generated at your SolarEdge Site in Wh
-    energy_this_year:
-      description: Energy generated this year at your SolarEdge Site in Wh
-    energy_this_month:
-      description: Energy generated this month at your SolarEdge Site in Wh
-    energy_today:
-      description: Energy generated today at your SolarEdge Site in Wh
-    current_power:
-      description: Current generated power in W
-    site_details:
-      description: Site details such as name, location, status, etc.
-    meters:
-      description: Site equipment details of connected meters. The sensor value shows the number of meters. Each meter is available as an attribute
-    sensors:
-      description: Site equipment details of connected sensors. The sensor value shows the number of sensors. Each sensor is available as an attribute
-    gateways:
-      description: Site equipment details of connected gateways. The sensor value shows the number of gateways. Each gateway is available as an attribute
-    batteries:
-      description: Site equipment details of connected batteries. The sensor value shows the number of batteries. Each battery is available as an attribute
-    inverters:
-      description: Site equiment details of connected inverters. The sensor values shows the number of inverters. Each inverter is available as an attribute
-    power_consumption:
-      description: Show the current power consumption of your SolarEdge Site (additional hardware required)  
-    solar_power:
-      description: Show the current power production of your SolarEdge Site (additional hardware required)
-    grid_power:
-      description: Show the current power flow from/to the power grid. State will be either positive of negative indicating importing or exporting electricity (additional hardware required)
-    storage_power:
-      description: Show the current power flow from/to your site storage. State will be either positive of negative indicating discharging or charging (additional hardware required)
-
 {% endconfiguration %}
 
-If no **monitored_conditions** are specified, only **current_power** will be enabled.
-
-### Full configuration sample
+## Full configuration example
 
 A full configuration entry would look like the sample below.
 
 {% raw %}
 ```yaml
 # Example configuration.yaml entry
-sensor:
-  - platform: solaredge
-    api_key: API_KEY
-    site_id: SITE_ID
-    name: SolarEdge
-    monitored_conditions:
-      - current_power
-      - energy_today
-      - energy_this_month
-      - energy_this_year
-      - lifetime_energy
-      - site_details
-      - meters
-      - sensors
-      - gateways
-      - batteries
-      - inverters
-      - power_consumption
-      - solar_power
-      - grid_power
-      - storage_power
+solaredge:
```

Comment:
I think we can skip this full example

Reply:
Agreed, it doesn't add much now anymore. Removed that one.